Content.Shared/Wall/ShuttleWallVisuals.cs

```diff
@@ -0,0 +1,10 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Wall
+{
+    [Serializable, NetSerializable]
+    public enum ShuttleWallVisuals
+    {
+        DeconstructionStage,
+    }
+}
```

Comment:
Does this need a new byte over reinforcedwallvisuals or the likes?

Also file-scoped namespace.

Reply:
>Does this need a new byte over reinforcedwallvisuals or the likes?

If you have a better suggestion, I'm open to it? This just worked....

Fixed namespace.